frameworks/rooch-framework/sources/auth_validator/bitcoin_validator.move

```diff
@@ -26,6 +30,49 @@ module rooch_framework::bitcoin_validator {
         BITCOIN_AUTH_VALIDATOR_ID
     }
 
+    public entry fun rotate_authentication_key_entry(
+        account: &signer,
+        public_key: vector<u8>
+    ) {
+        // compare newly passed public key with Ethereum public key length to ensure it's compatible
+        assert!(
+            vector::length(&public_key) == ecdsa_k1::public_key_length(),
+            ErrorInvalidPublicKeyLength
+        );
+
+        // User can rotate the authentication key arbitrarily, so we do not need to check the new public key with the account address.
+        let authentication_key = public_key_to_authentication_key(public_key);
+        let account_addr = signer::address_of(account);
+        rotate_authentication_key(account_addr, authentication_key);
```

Comment:
should we support rotate auth key to bitcoin account?

Reply:
It feels unnecessary at the moment，I don't know if there'll be a scene, and there's no security risk